### PR TITLE
13386 Add CEQR to DSOW show page

### PR DESCRIPTION
**Pull request by CeciliaCY**
### Summary
13386 Display the CEQR Questionnaire on the submitted DSOW form

#### Task/Bug Number
[Fixes AB#13386](https://dcp-paperless.visualstudio.com/ZAP%20Portals/_workitems/edit/13386)

### Technical Explanation
Add CEQR section in show.hbs based on edit.hbs. Also add SaveableForm to enable dynamic nav on the right. Similar to #895

### CODE REVIEW

File: client/app/components/packages/scope-of-work-draft/show.hbs [hunk 1]
```diff
@@ -1,71 +1,139 @@
...
-        data-test-package-statuscode
-      >
-        {{optionset 'bbl' 'boroughs' 'label' @package.project.dcpBorough}} |
-        {{optionset 'package' 'statuscode' 'label' @package.statuscode}}
-      </p>
-    </section>
+      {{#if (gte saveableForm.data.ceqrInvoiceQuestionnaires.length 1)}}
+          {{#let saveableForm.data.singleCeqrInvoiceQuestionnaire as |ceqrInvoiceQuestionnaire|}}
+            <saveableForm.Section
+            @title="Ceqr Invoice Questionnaire"
```
Comment (godfreyyeung) [inline client/app/components/packages/scope-of-work-draft/show.hbs:45]: tiny nit: indent the attributes here

File: client/app/components/packages/scope-of-work-draft/show.hbs [hunk 2]
```diff
@@ -1,71 +1,139 @@
...
+      {{#if (gte saveableForm.data.ceqrInvoiceQuestionnaires.length 1)}}
+          {{#let saveableForm.data.singleCeqrInvoiceQuestionnaire as |ceqrInvoiceQuestionnaire|}}
+            <saveableForm.Section
+            @title="Ceqr Invoice Questionnaire" 
+            @elementId="ceqr-invoice-questionnaire"
+            >
+              <saveableForm.SaveableForm
+                @model={{ceqrInvoiceQuestionnaire}}
+                @validators={{array (hash) (hash)}} 
+              >
```
Comment (godfreyyeung) [inline client/app/components/packages/scope-of-work-draft/show.hbs:51]: Why is this necessary? It doesn't look like it is being used. There is no yield like the first SaveableForm above. I.e. no `as |cequirInvoiceSaveableForm|`

I think this was only necessary on the Edit page, but doesn't look necessary here

File: client/app/components/packages/scope-of-work-draft/show.hbs [hunk 3]
```diff
@@ -1,71 +1,139 @@
...
+              </small>
+            </li>
+          {{/each}}
+        </ul>
+      </saveableForm.Section>
+    </div>
+
+    <div class="cell large-4 sticky-sidebar">
+      <Messages::Assistance class="large-margin-top" />
+      <saveableForm.PageNav>
```
Comment (godfreyyeung) [inline client/app/components/packages/scope-of-work-draft/show.hbs:134]: Since there is no content inside this component, I think you can just write
`<saveableForm.PageNav />` 

instead of the opening and closing tags 
```
  <saveableForm.PageNav>
  </saveableForm.PageNav>
```

**Review by godfreyyeung — COMMENTED** [review]
Thanks! So fast. I wrote a question and a few comments

File: client/app/components/packages/scope-of-work-draft/show.hbs (hunk 2 above)
Comment (CeciliaCY) [reply client/app/components/packages/scope-of-work-draft/show.hbs]: To bring in ceqrInvoiceQuestionnaire. Maybe you have a better way.

File: client/app/components/packages/scope-of-work-draft/show.hbs (hunk 3 above)
Comment (CeciliaCY) [reply client/app/components/packages/scope-of-work-draft/show.hbs]: Fixed

File: client/app/components/packages/scope-of-work-draft/show.hbs (hunk 2 above)
Comment (godfreyyeung) [reply client/app/components/packages/scope-of-work-draft/show.hbs]: It should still work without this. 
When you display a value like on line 57, it is referencing the  `ceqrInvoiceQuestionnaire` block variable from line 43. 

Previously on the Edit page, there is a new `saveableForm.SaveableForm` component in order to render Input fields, etc, but now that's not necessary because the user already inputted the data

File: client/app/components/packages/scope-of-work-draft/show.hbs (hunk 1 above)
Comment (CeciliaCY) [reply client/app/components/packages/scope-of-work-draft/show.hbs]: Corrected

File: client/app/components/packages/scope-of-work-draft/show.hbs (hunk 2 above)
Comment (CeciliaCY) [reply client/app/components/packages/scope-of-work-draft/show.hbs]: Thanks @godfreyyeung , corrected and also added a test case.

**Review by godfreyyeung — APPROVED** [review]
